src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/InfoBarTemplateSettings.cs

```diff
@@ -1,23 +1,38 @@
-﻿// MUX reference InfoBarTemplateSettings.properties.cpp, commit 3125489
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// MUX reference InfoBarTemplateSettings.properties.cpp, commit 3125489
 
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
 
 namespace Microsoft.UI.Xaml.Controls
 {
+	/// <summary>
+	/// Provides calculated values that can be referenced as TemplatedParent
+	/// sources when defining templates for an InfoBar.
+	/// </summary>
 	public partial class InfoBarTemplateSettings : DependencyObject
     {
+		/// <summary>
+		/// Initializes a new instance of the InfoBarTemplateSettings class.
+		/// </summary>
 		public InfoBarTemplateSettings()
 		{
 		}
 
+		/// <summary>
+		/// Gets the icon element.
+		/// </summary>
 		public IconElement IconElement
 		{
 			get => (IconElement)GetValue(IconElementProperty);
 			set => SetValue(IconElementProperty, value);
 		}
 
-		public static readonly DependencyProperty IconElementProperty =
+		/// <summary>
+		/// Identifies the InfoBarTemplateSettings.IconElement dependency property.
+		/// </summary>
+		public static DependencyProperty IconElementProperty { get; } =
```

Comment:
This also should have been caught by symbol matching.

Reply:
Oh, actually symbol matching **did** catch this one. It's the PR that's not merged yet: https://github.com/unoplatform/uno/pull/7249/files.

However, the namespace doesn't seem to be caught. So we may want some adjustments.